hw/ip/power_manager/data/power_manager.hjson

```diff
@@ -395,7 +395,7 @@
       swaccess: "rw",
       hwaccess: "hro",
       fields: [
-        { bits: "0", name: "EN_WAIT_FOR_INTR", desc: "Enable wait for interrupt" }
+        { bits: "31:0", name: "EN_WAIT_FOR_INTR", desc: "Enable wait for interrupt" }
```

Comment:
if there are 8 events, it is probably clear to leave this equal to 8bit (or whatever the number)

Reply:
Solved.